static/js/utils/disaster_event_map_utils.tsx

```diff
@@ -262,12 +262,8 @@ function fetchEventPoints(
         if (eventTypeSpec.displayProp) {
           for (const dp of eventTypeSpec.displayProp) {
             if (dp.prop in eventData.propVals) {
-              const val = parseEventPropVal(
-                eventData.propVals[dp.prop].vals,
-                dp.unit || "",
-                reqParams
-              );
-              if (!isNaN(val)) {
+              const val = eventData.propVals[dp.prop].vals[0];
```

Comment:
check that vals is non empty before accessing the first element? Or just use the `getValue` util function to get the value with all of this handled

Reply:
Added check for non-empty, `getValue()` expects a list of nodes as input instead.